packages/blast-utilities/run-docker-commands.js

```diff
@@ -28,12 +30,14 @@ const executeRun = function(arg) {
   runCommand(DOCKER_RUN_CMD + arg)
 }
 
-const executeNode = function(arg) {
-  runCommand(dockerComposeCmd + NODE_CMD + arg)
+const executeNode = function(arg, enableTty = true) {
```

Comment:
Is it a better idea to have two separate functions instead of one with a parameter as no yargs command options related to TTY use executeNode and TTY varies only for code purpose (and also commands to docker are a little different)?